packages/design-system/src/styles/settings/variables/_layout.scss

```diff
@@ -1,24 +1,36 @@
 $multiple: 8px !default;
 
+// $spacers defines all spacer units. The key (in quotes)
+// is the name appended to the generated utility classes,
+// i.e., `ds-u-sm-padding--05`, and the value ($multiple * X)
+// is what that space evaluates as (in `px`).
 $spacers: (
-  0,
-  $multiple,
-  $multiple * 2,
-  $multiple * 3,
-  $multiple * 4,
-  $multiple * 5,
-  $multiple * 6,
-  $multiple * 7
+  '0': 0,
+  '05': $multiple * 0.5,
+  '1': $multiple,
+  '2': $multiple * 2,
+  '3': $multiple * 3,
+  '4': $multiple * 4,
+  '5': $multiple * 5,
+  '6': $multiple * 6,
+  '7': $multiple * 7,
 ) !default;
 
-$spacer-1: nth($spacers, 2) !default; // 8px
-$spacer-2: nth($spacers, 3) !default; // 16px
-$spacer-3: nth($spacers, 4) !default; // 24px
-$spacer-4: nth($spacers, 5) !default; // 32px
-$spacer-5: nth($spacers, 6) !default; // 40px
-$spacer-6: nth($spacers, 7) !default; // 48px
-$spacer-7: nth($spacers, 8) !default; // 56px
-$spacer-half: $spacer-1 / 2; // 4px
+// Sass variables are generated from $spacers-values (needed
+// due to a different naming syntax between util classes
+// and Sass variables).
+$spacers-values: map-values($spacers);
+
+// 🚨 Note: If you change the order in $spacers, you may
+// need to update the index used to generate the variables
+$spacer-half: nth($spacers-values, 2) !default; // 4px
+$spacer-1: nth($spacers-values, 3) !default; // 8px
+$spacer-2: nth($spacers-values, 4) !default; // 16px
+$spacer-3: nth($spacers-values, 5) !default; // 24px
+$spacer-4: nth($spacers-values, 6) !default; // 32px
+$spacer-5: nth($spacers-values, 7) !default; // 40px
+$spacer-6: nth($spacers-values, 8) !default; // 48px
+$spacer-7: nth($spacers-values, 9) !default; // 56px
```

Comment:
@ericwbailey @line47 lemme know if I borked your sass vars up too much. I thought this might be an easier way to add/remove util spacers in the future.